src/Languages/Core/Test/Braces/BraceTokenizerTest.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using FluentAssertions;
+using Microsoft.Languages.Core.Braces;
+using Microsoft.UnitTests.Core.XUnit;
+using Xunit;
+
+namespace Microsoft.Languages.Core.Test.Braces {
+    [ExcludeFromCodeCoverage]
+    [Category.Languages.Core]
+    public sealed class BraceTokenizerTest {
+        [CompositeTest]
+        [InlineData("", new BraceTokenType[0])]
+        [InlineData("func()[x]{}", new[] { BraceTokenType.OpenBrace, BraceTokenType.CloseBrace, BraceTokenType.OpenBracket, BraceTokenType.CloseBracket, BraceTokenType.OpenCurly, BraceTokenType.CloseCurly })]
```

Comment:
you can add `using static BraceTokenType` to avoid adding `BraceTokenType` for every enum value